app/routes/users.routes.js

```diff
@@ -8,6 +8,14 @@ module.exports = function (app) {
     // TODO add a PUT at this route to update currently logged in user
     // So, you change it to app.route('/users/from-token').get(users.getUserFromToken).put(users.someFunctionName);
     app.get('/users/from-token', users.getUserFromToken);
+  
+    app.post('/user/update-first-name', users.updateFirstName);
```

Comment:
To better match REST API naming conventions change this line to:

`app.put('/users/from-token/first-name', users.updateFirstName);`